pillar/heroku/mitxonline.sls

```diff
@@ -0,0 +1,119 @@
+{% set minion_id = salt.grains.get('id', '') %}
+{% set environment = minion_id.split('-')[-1] %}
+{% set rds_endpoint = salt.boto_rds.get_endpoint('production-apps-rds-postgres-mitxonline') %}
+
+{% set env_dict = {
+    'ci': {
+      'app_name': 'mitxonline-ci',
+      'env_name': 'ci',
+      'GOOGLE_TRACKING_ID': '',
+      'GOOGLE_TAG_MANAGER_ID': '',
+      'release_branch': 'master',
+      'app_log_level': 'INFO',
+      'sentry_log_level': 'ERROR',
+      'OPENEDX_API_BASE_URL': '',
+      'openedx_environment': 'mitxonline-qa',
+      'MAILGUN_FROM_EMAIL': 'MITx Online <no-reply@mitxonline-ci-mail.mitxonline.mit.edu>',
+      'MAILGUN_SENDER_DOMAIN': 'mitxonline-ci-mail.mitxoline.mit.edu',
+      'MITXONLINE_BASE_URL': 'https://mitxonline-ci.mitxonline.mit.edu',
+      'MITXPRO_SECURE_SSL_HOST': 'mitxonline-ci.mitxonline.mit.edu',
+      'vault_env_path': 'rc-apps',
+      },
+    'rc': {
+      'app_name': 'mitxonline-rc',
+      'env_name': 'rc',
+      'GOOGLE_TRACKING_ID': '',
+      'GOOGLE_TAG_MANAGER_ID': '',
+      'release_branch': 'master',
+      'app_log_level': 'INFO',
+      'sentry_log_level': 'ERROR',
+      'OPENEDX_API_BASE_URL': '',
+      'openedx_environment': 'mitxonline-qa',
+      'MAILGUN_FROM_EMAIL': 'MITx Online <no-reply@mitxonline-rc-mail.mitxonline.mit.edu>',
+      'MAILGUN_SENDER_DOMAIN': 'mitxonline-rc-mail.mitxoline.mit.edu',
+      'MITXONLINE_BASE_URL': 'https://mitxonline-rc.mitxonline.mit.edu',
+      'MITXPRO_SECURE_SSL_HOST': 'mitxonline-rc.mitxonline.mit.edu',
+      'vault_env_path': 'rc-apps',
+      },
+    'production': {
+      'app_name': 'mitxonline-production',
+      'env_name': 'ci',
+      'GOOGLE_TRACKING_ID': '',
+      'GOOGLE_TAG_MANAGER_ID': '',
+      'release_branch': 'master',
+      'app_log_level': 'INFO',
+      'sentry_log_level': 'ERROR',
+      'OPENEDX_API_BASE_URL': '',
+      'openedx_environment': 'mitxonline-production',
+      'MAILGUN_FROM_EMAIL': 'MITx Online <no-reply@mitxonline.mit.edu>',
+      'MAILGUN_SENDER_DOMAIN': 'mitxoline.mit.edu',
+      'MITXONLINE_BASE_URL': 'https://mitxonline.mit.edu',
```

Comment:
We should determine if this is intended to be the production domain, because if so then we will require setting up the DNS through Fastly to use their Anycast IP as the target of an `A` record.

Reply:
Assume yes, but there's still a decent change the name will change. If is saves you time, we can go with something else for now.